### PR TITLE
optimize

### PR DESCRIPTION
**Pull request by DoraNebula**
use switch case to replace if in some cases.
reorganize code of aria client & aria service, optimize data struct of feed back from aria
add bid configuration
when start running:
1. read bid configuration for miner from api
2. compare the local bid configuration and those got from api
3. if different, then update local bid configuration to server through api

### CODE REVIEW

File: common/utils/aria2Client.go [hunk 1]
```diff
@@ -17,13 +17,69 @@ type Aria2Client struct {
...
 
+type Aria2DownloadOption struct {
+	Out string   `json:"out"`
+	Dir string   `json:"dir"`
+}
+
+type Aria2Status struct {
+	Id 		string               `json:"id"`
+	JsonRpc string               `json:"jsonrpc"`
+	Error 	*Aria2Error          `json:"error"`
```
Comment (zzhao-nbai) [inline common/utils/aria2Client.go:35]: format Format alignment

File: common/utils/aria2Client.go [hunk 2]
```diff
@@ -17,13 +17,69 @@ type Aria2Client struct {
...
+	Result 	*Aria2StatusResult   `json:"result"`
+}
+
+type Aria2Download struct {
+	Id 		string               `json:"id"`
+	JsonRpc string               `json:"jsonrpc"`
+	Error 	*Aria2Error          `json:"error"`
```
Comment (zzhao-nbai) [inline common/utils/aria2Client.go:42]: Format alignment

File: common/utils/swanClient.go [hunk 3]
```diff
@@ -32,13 +32,17 @@ type OfflineDealData struct {
...
-func GetSwanClient() (*SwanClient) {
+func GetSwanClient() *SwanClient {
 	mainConf := config.GetConfig().Main
 	uri := mainConf.SwanApiUrl +"/user/api_keys/jwt"
 	data := TokenAccessInfo{ApiKey: mainConf.SwanApiKey, AccessToken: mainConf.SwanAccessToken}
 	response := HttpPostNoToken(uri, data)
 
 	jwtToken := GetFieldMapFromJson(response,"data")
+	if jwtToken == nil {
+		logs.GetLogger().Error("Error: fail to get jwt")
```
Comment (zzhao-nbai) [inline common/utils/swanClient.go:43]: Add return, otherwise the analysis below will report an error

File: common/utils/aria2Client.go (hunk 2 above)
Comment (DoraNebula) [reply common/utils/aria2Client.go:42]: fixed

File: common/utils/swanClient.go (hunk 3 above)
Comment (DoraNebula) [reply common/utils/swanClient.go]: fixed

File: common/utils/aria2Client.go (hunk 1 above)
Comment (DoraNebula) [reply common/utils/aria2Client.go:35]: fixed

**Pull request merged**